lib/index.js

```diff
@@ -37,11 +38,9 @@ exports.register = function (server, options, next) {
     apply: true
   })
 
-  if (!settings.disableHapiDecorate) {
-    server.decorate('server', 'action', internals.action(server))
-    server.decorate('reply', 'act', internals.replyAct)
-    server.handler('act', internals.handlerAct)
-  }
+  server.decorate('server', [options.hapiDecoratePrefix || '', 'action'].join(''), internals.action(server))
```

Comment:
I agree, but what happens if no prefix is defined. then it turns into request._act or reply._act.